g_docs.go

```diff
@@ -592,6 +593,9 @@ func parserComments(comments *ast.CommentGroup, funcName, controllerName, pkgpat
 					case "thrift_webcontent_json":
 						opts.Consumes = append(opts.Consumes, content_type_thrift_json_webcontent_v1)
 						opts.Produces = append(opts.Produces, content_type_thrift_json_webcontent_v1)
+					case "form":
+						opts.Consumes = append(opts.Consumes, aform)
+						opts.Produces = append(opts.Produces, aform)
```

Comment:
So just for my understanding, if we specify `formData`, the we set the consumes (and produces) property here fixed,  although we might "forgotten" it (because we dont know better & using the beego doc wrong)?

Reply:
I think when we fork this repo, bee tool just have no support for / haven't handled this yet. and our swagger web is also not showing any errors either event though the format is wrong 😓 . the newer bee version includes this `form` as well in their [code](https://github.com/beego/bee/blob/0d739375d1d89138b21184f1c9f3af115f8f4622/generate/swaggergen/g_docs.go#L733).

Reply:
Ah wow good to know 👍